extensions/amp-analytics/OWNERS

```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{ name: 'ampproject/wg-analytics' }, { name: 'avimehta' }]
+      owners: [{ name: 'ampproject/wg-analytics' }, { name: '?avimehta' }]
```

Comment:
Per [the example file](https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example), the new JSON format uses explicit options instead of the rather opaque `?`/`#` modifiers. This line should be:

```javascript
owners: [
  { name: 'ampproject/wg-analytics' },
  { name: 'avimehta', requestReviews: false}
]
```

Reply:
Ah, thanks for referring to the new format.  I agree the new format is better 👍 
Fixed.